students/AndyKwok/lesson09/mailroom_pt9.py

```diff
@@ -0,0 +1,148 @@
+# Description: Mailroom Program - OOP
+# Author: Andy Kwok
+# Last Updated: 9/18/18
+# ChangeLog: 
+# v1.0 - Initialization
+
+#!/usr/bin/env python3
+ 
+
+class Donor():
+    ''' Class to hold donor name and donation information'''
+    def __init__(self, First = None, Last = None):
+        self.__First = First
+        self.__Last = Last
+        self.Donation_Record = []
+        self.__Fullname = None
```

Comment:
all of your var names should be lower case:
```
def __init__(self, first=None, last=None):
    self.first = first
    self.last = last
....
```
also, there is usually no need for vars to be private - so no need for double under scores.

There are cases when it is useful but not here.